.gitignore

```diff
@@ -0,0 +1,109 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I would prefer to avoid this "default" ignore from the commit history.
But leave it be, take that into account for the next time 👍